### ISSUE TITLE
Accessibilty of examples (e. g. RadzenPanel)

### ISSUE THREAD

**Issue by efinder2** [radzenhq/radzen-blazor#2596]
**Is your feature request related to a problem? Please describe.**
Examples in the Documentation violate WCAG 1.3.1. They use semantic to generate visual representation. For example in the Demo for the penal uses many `h6` to get a small bold text. The heading level above it is `h1`.

While exploring the components I copied the example code. Our automated accessiblity tests flagged them instantly as wrong. According to the accessibilty section of documentation  I expected the components provide "an inclusive user experience out of the box".

**Describe the solution you'd like**
All examples full fill all wcag AA criteria in context of the page. Ideally they work in many scenarios when copied to the user code. Document warnings for possbile violations and pitfalls might occur when doing this.

Additionally notices explain how to use, if possible, the component in a WCA AAA compliant way.

**Describe alternatives you've considered**

**Additional context**
Our application is required by law to fullfill the EN 301 549. This has a lot in common with the WCAG 2.0 AA.
